src/Credentials/CredentialProvider.php

```diff
@@ -88,6 +89,11 @@ public static function defaultProvider(array $config = [])
             !isset($config['use_aws_shared_config_files'])
             || $config['use_aws_shared_config_files'] !== false
         ) {
+            $defaultChain['sso'] = self::sso(
```

Comment:
This introduces a new PHP notice when there is no `[profile default]` section in the ~/.aws./config file, which should be always, since the default section is supposed to be `[default]`: